build/.goreleaser.yml

```diff
@@ -0,0 +1,114 @@
+before:
+  hooks:
+  # You may remove this if you don't use go modules.
+  #- go mod download
+  # you may remove this if you don't need go generate
+  #- go generate ./...
+  #- make validate
+
+builds:
+  - id: nri-redis-nix
+    main: ./src
+    binary: nri-redis
+    gobinary: "go1.9.7"
+    #ldflags:
+    #  - -s -w -X github.com/newrelic/nri-flex/internal/load.IntegrationVersion={{.Version}}
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+
+  - id: nri-redis-win
+    main: ./src
+    binary: nri-redis
+    gobinary: "go1.9.7"
+    #ldflags:
+    #  - -s -w -X github.com/newrelic/nri-flex/internal/load.IntegrationVersion={{.Version}}
+    goos:
+      - windows
+    goarch:
+      - 386
+      - amd64
+    hooks:
+      pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "redis"
+
+nfpms:
+  - id: nri-redis-deb
+    builds:
+      - nri-redis-nix
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure redis Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from redis components."
+    license: "https://newrelic.com/terms (also see LICENSE.txt installed with this package)"
```

Comment:
I think the LICENSE file doesn't have any extension

Reply:
btw, the windows package doesn't contain the LICENSE file. It would be really nice to add it there as well.